website/docs/dbt-cli/configure-your-profile.md

```diff
@@ -89,7 +89,7 @@ dbt supports multiple targets within one profile to encourage the use of separat
 
 A typical profile for an analyst using dbt locally will have a target named `dev`, and have this set as the default.
 
-You may also have a `prod` target within your profile, which creates the objects in your production schema. However, since it's often desirable to perform production runs on a schedule, we recommend deploying your dbt project on a separate machine to your local machine. As such, most dbt users will only have a dev target in their profile on their local machine.
+You may also have a `prod` target within your profile, which creates the objects in your production schema. However, since it's often desirable to perform production runs on a schedule, we recommend deploying your dbt project on a separate machine from your local machine. As such, most dbt users will only have a dev target in their profile on their local machine.
```

Comment:
```suggestion
You may also have a `prod` target within your profile, which creates the objects in your production schema. However, since it's often desirable to perform production runs on a schedule, we recommend deploying your dbt project to a separate machine other than your local machine. Most dbt users only have a `dev` target in their profile on their local machine.
```